### DISCUSSION TITLE
UX Patterns for Devs using Nextra 4 app router

### DISCUSSION THREAD

**Discussion by thedaviddias** [shuding/nextra#3801] [Show and tell]
Hey @shuding!

I've recently launched a new open-source project: https://github.com/thedaviddias/ux-patterns-for-developers using Nextra latest [nextra@4.0.0-app-router.34](https://github.com/shuding/nextra/releases/tag/nextra%404.0.0-app-router.34).

Didn't encounter too many problems, just some issues with Tailwind CSS 4 beta. I had to add `@variant dark (&:where(.dark, .dark *));` to avoid some issues around the `prefers-color-scheme`.

Let me know if you need any help or support, will be happy to help with Nextra!

**Comment by rafalzawadzki** [comment]
@thedaviddias thanks for sharing! I'm going to use your project as a starter for mine :) wanted to try 4.0 but wasn't sure if it can be used at all.

**Reply by thedaviddias** [reply]
Let me know how it goes!

**Reply by rafalzawadzki** [reply]
it went smoothly! very good example for getting started with nextra + app router. 
I made this documentation page: https://supadata.ai

Maybe one question in case you know! I noticed that in your project the sidebar does not show background highlight for currently active page:
![CleanShot 2024-12-15 at 11  54 13@2x](https://github.com/user-attachments/assets/91b301a5-5a85-4358-b05d-537326b5e3f2)
![CleanShot 2024-12-15 at 11  54 44@2x](https://github.com/user-attachments/assets/1f9b9bce-ad77-4e5e-9359-4e1e7a90e4ef)

Only relevant info I could find is here: https://nextra.site/docs/docs-theme/theme-configuration#theme-color but does not seem like you purposefully disabled this detail in your project?
